db/keyval/etcdv3/plugin_impl.go

```diff
@@ -88,10 +93,17 @@ func (p *Plugin) Init() error {
 		return err
 	}
 
-	// register for providing status reports (polling mode)
+	// Init ETCD transport
+	etcdTransport, err := p.InitTransport(p.Skeleton.Logger)
+	if err != nil {
+		return err
+	}
+	p.Transports.RegisterEtcdTransport(etcdTransport)
```

Comment:
Let's isolate transport to Flavor. MUX does not need to know about registration of transports anymore. Just inject for example to dbsync the db/keyval/plugin_api.go